doc/modules/ROOT/pages/graph-object.adoc

```diff
@@ -163,6 +160,16 @@ If multiple node dataframes are used, they need to contain distinct node ids acr
 * Prior to the `construct` call, a call to `GraphDataScience.set_database` must have been made to explicitly specify which Neo4j database should be targeted.
 
 
+=== Limitations on community edition
+
+For users of GDS community edition, `gds.graph.construct` leverages Cypher Aggregation in implementation.
```

Comment:
Not sure people would understand what "cypher aggregation" is.
Also I think these docs are relevant for all dataloading (construct, OGB, NX, pre-canned, etc). I would if we should have a shared note we can include or something